BLSQ/iaso
# POLIO-1506  POLIO-1507 Move scope tab to round

Merge the tabs “Rounds” and “Scope” into one

Evaluation tab will contain a lot of the fields that are currently standing in “Rounds” tab:

- District Passing LQAS (automatic results computation)
- District Failing LQAS (automatic results computation)
- LQAS start date - for this one, careful that we have a redirection from LQAS Country page to encode the start and end dates)
- LQAS end date - for this one, careful that we have a redirection from LQAS Country page to encode the start and end dates)
- IM start date
- IM end date
- Main reason for non vaccination
- % children missed IN households
- % children missed OUT OF households
- % children missed IN + OUT OF households
- % awareness campaign planning

Just like Rounds tab, there should be entries for each Round (so there should be Round tabs)

Related JIRA tickets : POLIO-1506,  POLIO-1507

## Self proofreading checklist



## Changes

- remove scope tab and use scope field on round tab
- add evaluation tab for specific polio rounds infos
- relayout a bit sub tabs

## How to test

Create or edit a campaign, go to round tab and play with the scope / fields
Go to evaluation tab, play with fields and try to save.

Double check that your change has been saved

## Print screen / video
![Screenshot 2024-03-19 at 10 12 41](https://github.com/BLSQ/iaso/assets/12494624/61d06ab5-c625-4e67-8313-d8d182b0fcb6)
![Screenshot 2024-03-19 at 10 12 29](https://github.com/BLSQ/iaso/assets/12494624/b70a04a8-9ed0-49c1-a65e-4292bef29601)
